share/benchmark/xtp_benchmark.in

```diff
@@ -97,83 +113,102 @@ progargs.add_argument('-r', '--run',
                       default=0,
                       help='Run benchmark')
 
-progargs.add_argument('-a', '--analyze',
+progargs.add_argument('-s', '--setup',
                       action='store_const',
                       const=1,
                       default=0,
-                      help='Show results')
+                      help='Setup Benchmark')
 
-progargs.add_argument('--reset',
-                      action='store_const',
-                      const=1,
-                      default=0,
-                      help='Delete all benchmark run data')
+progargs.add_argument('-i', '--input_folder',
+                      required=False,
+                      type=str,
+                      default="Geometries/Geometries_ThielSet",
+                      help='Folder to take geometries from')
+
+progargs.add_argument('-o', '--output_folder',
+                      required=False,
+                      type=str,
+                      default="RUN",
+                      help='Folder to write benchmark to')
 
-progargs.add_argument('--update',
+progargs.add_argument('-a', '--analyze',
                       action='store_const',
                       const=1,
                       default=0,
-                      help='Update references')
+                      help='Graphically analyse results from outputfolder')
+
+progargs.add_argument('--reference',
+                      required=False,
+                      type=str,
+                      help='In combination with analyze options compare to a reference')
+
+HasGPUs = HasGPUs()
+if HasGPUs:
+    progargs.add_argument('-g', '--gpus',
+                          required=False,
+                          type=int,
+                          default=0,
+                          help='Number of GPUs to use')
 
 
 OPTIONS = progargs.parse_args()
 
-if OPTIONS.reset:
-    print("Deleting 'RUN' directory")
-    try:
-        shutil.rmtree("RUN")
-    except OSError as e:
-        print("Error: %s - %s." % (e.filename, e.strerror))
+
+if OPTIONS.setup:
+    print("Creating benchmark directory: {}".format(OPTIONS.output_folder))
+    if os.path.isdir(OPTIONS.output_folder):
+        xxquit("Folder '{}' already exists".format(OPTIONS.output_folder))
+    os.mkdir(OPTIONS.output_folder)
+
+    input_molecules = []
+
+    pattern = "*.xyz"
+
+    for path, subdirs, files in os.walk(OPTIONS.input_folder):
+        for name in files:
+            if fnmatch.fnmatch(name, pattern):
+                input_molecules.append([path, name])
+    
+   
+    for mol in input_molecules:
+        newpath=os.path.join(OPTIONS.output_folder,os.path.splitext(mol[1])[0])
```

Comment:
Instead of representing paths with *strings* you can also use the `Path` class from [pathlib](https://docs.python.org/3/library/pathlib.html). Then you can do things like:
```py
newpath = Path(OPTIONS.output_folder) / Path(mol[1]).stem
```
You can even convert `OPTIONS.output_folder` to `Path` since you read it, but that is up to your preference :)

Reply:
That is a good idea. :) I will do this after the next round of review